.tzapignore

```diff
@@ -23,4 +23,5 @@ build
 *graphviz*
 *generated*
 *actionpb*
-tfs.go
\ No newline at end of file
+tfs.go
+.next
```

Comment:
This is not the right file to edit. Please edit cli/cmd/cmdutil/fileevaluator/fileevaluator.go to edit the base tzapignore

Reply:
Thank you, I will apply the change there